vhdbuilder/packer/install-dependencies.sh

```diff
@@ -257,6 +257,32 @@ EOF
 
 echo "${CONTAINER_RUNTIME} images pre-pulled:" >> ${VHD_LOGS_FILEPATH}
 
+
+downloadArtifactStreamingComponents() {
+  pushd /tmp || exit $ERR_MIRROR_PROXY_INSTALL_ERR
+  # download acr-mirror proxy
+  MIRROR_PROXY_VERSION='0.1.0-dev-19'
+  MIRROR_PROXY_URL="https://github.com/juliusl/lifec_registry/releases/download/v${MIRROR_PROXY_VERSION}/acr-mirror-1804-v${MIRROR_PROXY_VERSION}.deb"
+  wget $MIRROR_PROXY_URL
+  apt_get_install 30 1 600 "./acr-mirror-1804-v${MIRROR_PROXY_VERSION}.deb" || exit $ERR_MIRROR_PROXY_DOWNLOAD_ERR
+  rm "./acr-mirror-1804-v${MIRROR_PROXY_VERSION}.deb"
+  popd || exit $ERR_MIRROR_PROXY_INSTALL_ERR
+
+  sudo apt install libnl-3-dev libnl-genl-3-dev -y
+  sudo /opt/acr/tools/overlaybd/install.sh || exit $ERR_MIRROR_PROXY_INSTALL_ERR
+  sudo /opt/acr/tools/overlaybd/enable-http-auth.sh || exit $ERR_MIRROR_PROXY_INSTALL_ERR
+  modprobe target_core_user || exit $ERR_MIRROR_PROXY_INSTALL_ERR
+  
+}
+
+enableArtifactStreamingServices() {
+  sudo /opt/acr/tools/overlaybd/enable.sh || exit $ERR_MIRROR_PROXY_INSTALL_ERR
+}
+
+if [[ "${CONTAINER_RUNTIME:=}" == "containerd"  && "${UBUNTU_RELEASE}" == "18.04" ]]; then
```

Comment:
I wouldn't bother with container runtime at this point, we should finish stripping that out of vhd build at least if not runtime (for old users)